avro-schema/sum-part.avsc

```diff
@@ -61,6 +61,17 @@
             "type": "long",
             "logicalType": "timestamp-millis",
             "doc": "time range information for the shares in this batch."
+        },
+        {
+            "name": "invalid_packet_uuids",
+            "type": {
+                "type": "array",
```

Comment:
As we (me and @tgeoghegan) talked in Slack, my concern with this is that it requires the entire list of uuids to be parsed whenever you want to read this header, it also prevents writing or reading this file effectively in a streaming way, since the entire message needs to be (de)serialized in memory at once. This is fine if we don't have many invalid UUIDs (and I don't think we expect that), however, it is a potential DoS vector (especially for the N-hourly reduction steps, which might be reading many of these files in serial without parallelization).

If we do it like this, then I think we should agree on truncating the number of items in this list to some maximum value. We can still keep track of the total number of failures in an overflow counter. Afaik this list is mostly intended as an eventual debugging aid?

@winstrom since you were the one who originally proposed this in the schema doc

Reply:
We could also put the invalid UUIDs back into a separate Avro encoded file (which is how it was laid out in the IDL doc) and instead put `invalid_packet_uuids_file_digest` here.

Reply:
Having the invalid packets in another file, hashed in the header, is a good design decision 👍  Thanks.

Reply:
I think that's a good idea, but I have a few more questions about error handling at all stages of the pipeline, which I am going to ask in an email to prio-implementers today in anticipation of resolving them during Thursday's meeting. We can implement whatever schema changes result from that conversation in a subsequent PR.